go/pgn/internal/pgncmn/common.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 ETH Zurich
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgncmn
+
+import (
+	"net"
+
+	"github.com/scionproto/scion/go/lib/addr"
+	"github.com/scionproto/scion/go/lib/env"
+	"github.com/scionproto/scion/go/lib/infra/infraenv"
+	"github.com/scionproto/scion/go/lib/infra/messenger"
+	"github.com/scionproto/scion/go/lib/infra/modules/itopo"
+	"github.com/scionproto/scion/go/lib/serrors"
+	"github.com/scionproto/scion/go/pgn/internal/pgnmsgr"
+	pgnconfig "github.com/scionproto/scion/go/pkg/pgn/config"
+)
+
+var (
+	IA    addr.IA
+	PLNIA addr.IA
+)
+
+func Init(cfg pgnconfig.PgnConf, sdCfg env.SCIONDClient, features env.Features) error {
+	router, err := infraenv.NewRouter(cfg.IA, sdCfg)
+	if err != nil {
+		return serrors.WrapStr("Error in Init mscmn", err)
+	}
+	nc := infraenv.NetworkConfig{
+		IA:                    cfg.IA,
+		Public:                &net.UDPAddr{IP: cfg.IP, Port: int(cfg.Port)},
+		SVC:                   addr.SvcPGN,
```

Comment:
Please lint the files in these PRs, at least gofmt and goimports

Reply:
This I had done, is there something specific that I am missing? I don't see any problems reported

Reply:
My bad, I had an old gofmt on the path from the debian package install of Go for Ubuntu 18.04:
```
circleci@golang:~/project/scion$ diff <(cat go/pgn/internal/pgncmn/common.go) <(/usr/lib/go-1.10/bin/gofmt go/pgn/internal/pgncmn/common.go)
41,43c41,43
<               IA:                    cfg.IA,
<               Public:                &net.UDPAddr{IP: cfg.IP, Port: int(cfg.Port)},
<               SVC:                   addr.SvcPGN,
---
>               IA:     cfg.IA,
>               Public: &net.UDPAddr{IP: cfg.IP, Port: int(cfg.Port)},
>               SVC:    addr.SvcPGN,
circleci@golang:~/project/scion$ diff <(cat go/pgn/internal/pgncmn/common.go) <(gofmt go/pgn/internal/pgncmn/common.go)
circleci@golang:~/project/scion$
```

I should have checked the actual output, there wasn't enough context to see that the current formatting is indeed correct.